src/comments.js

```diff
@@ -0,0 +1,93 @@
+const counterComment = (comments) => {
+  const result = comments ? `Comments(${comments.length})` : 'Comments (0)';
+  return result;
+};
+
+const getComments = async (id) => {
+  const apiUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/xDtzCPywXomy9vfRiKuT/comments?item_id=${id}`;
+  const response = await fetch(apiUrl, { method: 'GET' });
+  let comments = await response.json();
+  if (comments.error) {
+    comments = [];
+  }
+  return comments;
+};
+const postStrange = async (body) => {
+  const apiUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/xDtzCPywXomy9vfRiKuT/comments';
+  await fetch(apiUrl, {
+    method: 'POST',
+    body: JSON.stringify(body),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
```

Comment:
- [ ] Please make sure to validate the comments form for empty values, currently I can add an empty comment.